src/test/scala/firrtlTests/ExpandWhensSpec.scala

```diff
@@ -32,9 +32,51 @@ import org.scalatest._
 import org.scalatest.prop._
 import firrtl._
 import firrtl.passes._
+import firrtl.ir._
+import firrtl.Parser.IgnoreInfo
 
 class ExpandWhensSpec extends FirrtlFlatSpec {
+  def parse (input:String) = Parser.parse(input.split("\n").toIterator, IgnoreInfo)
```

Comment:
Style nit, should be 
```scala
private def parse(input: String) = ...
```
(Interestingly, I just noticed that Github syntax highlighting doesn't highlight the parameter name unless you put a space after the colon)